core/src/main/java/io/confluent/kafka/schemaregistry/storage/SchemaRegistryIdentity.java

```diff
@@ -99,6 +104,20 @@ public void setMasterEligibility(Boolean eligibility) {
     this.masterEligibility = eligibility;
   }
 
+  @JsonProperty(value = "scheme", defaultValue = "http")
+  public String getScheme() {
+    return scheme;
+  }
+
+  @JsonProperty(value = "scheme", defaultValue = "http")
+  public void setScheme(String scheme) {
+    this.scheme = scheme;
+  }
+
+  public static int getCurrentVersion() {
+    return CURRENT_VERSION;
```

Comment:
there's probably some docs that should be added explaining the upgrade path. keeping this version as is + defaulting to https if it is configured requires an upgrade path of full bounce to new version with only http support, then enable https support and bounce again.